config.yaml.sample

```diff
@@ -1,10 +1,12 @@
+queue:
+  type: file
+  name: queue
+store:
+  type: directory
+  name: tiles
+  path: osm
+  reduced-redundancy: true
```

Comment:
Can we have the sample config default to the types that we use in production, and have a comment for all the other types? Or maybe a separate commented line for each type?